W-17149740-update-kubernetes-cluster-requirements-kt (#349)
mulesoft/docs-private-cloud: modules/ROOT/pages/install-checklist.adoc

*** 3 K8s control plane nodes
*** 4 K8s worker nodes

> If you have Anypoint Platform PCE installed up to version 3.2.x, ensure you disable SWAP on all Anypoint Platform PCE nodes. Anypoint Platform PCE relies heavily on Kubernetes, which does not support the use of SWAP space. When SWAP is enabled, both Kubernetes and the SWAP daemon attempt to manage the same memory space, leading to CPU overload as they compete. To avoid this issue, do not to use SWAP space on machines running Anypoint Platform PCE.

Reviewer: ```suggestion
If you have Anypoint Platform PCE 3.2.x or earlier installed, ensure you disable SWAP on all Anypoint Platform PCE nodes. Anypoint Platform PCE relies heavily on Kubernetes, which doesn't support the use of SWAP space. When SWAP is enabled, Kubernetes and the SWAP daemon attempt to manage the same memory space, leading to CPU overload as they compete. To avoid this issue, don't use SWAP space on machines running Anypoint Platform PCE.
```
